src/gerbil/runtime/mop.ss

```diff
@@ -457,10 +464,9 @@ namespace: #f
                                 precedence-list slot-vector properties
                                 constructor* slot-table #f)))
 
-(def (class-precedence-list (klass : :class)) => :list
-  (cons klass (&class-type-precedence-list klass)))
+(def class-precedence-list &class-type-precedence-list)
```

Comment:
remove this if we are putting the class there.

Reply:
Not sure what you mean. Just replace uses of class-precedence-list by uses of class-type-precedence-list ?